modules/compute_metric.py

```diff
@@ -0,0 +1,143 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This example shows how to compute metric in multi-processing based on PyTorch native `DistributedDataParallel` module.
+It can even run on multi-nodes.
+Main steps to set up the distributed data parallel:
+
+- Execute `torch.distributed.launch` to create processes on every node for every process.
```

Comment:
Thanks for let me know this, in case to support PyTorch 1.5 - 1.8.1, I think we can still use `launch` here.
Will use `run` instead in future.

Thanks.